src/Home.js

```diff
@@ -1,17 +1,111 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View, FlatList, Pressable } from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
 
 export default function Home() {
+
+    const [todos, setTodos] = useState([]);
+
+    let subscription;
+
+    useEffect(() => {
+        onQuery = async () => {
+            const todos_list = await DataStore.query(Todo);
+            setTodos(todos_list);
+        };
+        onQuery();
+        if (typeof subscription === 'undefined') {
+            subscription = DataStore.observe(Todo).subscribe((msg) => {
```

Comment:
What are we doing with `msg` argument?

Reply:
the msg argument is the updated Todo object we receive whenever a change is reported back via the subscription. I copied the function from amplify js library docs.

Reply:
I do not see `msg` used any where here, would omitting prevent the callback from functioning correctly?

Reply:
Regarding msg, Imo it is a good idea to leave it there even though we are not using it. It will allow the user to easily know, they can receive the updated Todo Object here and add some more functionality if needed. We can mention this in the getting started guide which will be helpful I think.